Public/Src/Cache/DistributedCache.Host/Service/Internal/DistributedContentStoreFactory.cs

```diff
@@ -310,7 +311,7 @@ private async Task ApplySecretSettingsForLlsAsync(RedisContentLocationStoreConfi
 
             var eventStoreConfiguration = new EventHubContentLocationEventStoreConfiguration(
                 eventHubName: "eventhub",
-                eventHubConnectionString: GetRequiredSecret(secrets, _distributedSettings.EventHubSecretName),
+                eventHubConnectionString: (GetRequiredSecret(secrets, _distributedSettings.EventHubSecretName) as EventHubCredentials).ConnectionString,

```

Comment:
Not clear why you cast the result to a specific type in case of redis, but use 'as' cast here.